brink.sh

```diff
@@ -865,6 +880,12 @@ if [ "$COMMAND" = "purge" ] ; then
     exit 0
 fi
 
+if [ "$COMMAND" == "detect_os" ]; then
```

Comment:
need a comment to describe what this command does... or move it to a function and add comment to that function

Reply:
Added a comment. Thanks!